server/etcdserver/v3_server.go

```diff
@@ -271,6 +274,9 @@ func (s *EtcdServer) LeaseGrant(ctx context.Context, r *pb.LeaseGrantRequest) (*
 }
 
 func (s *EtcdServer) LeaseRevoke(ctx context.Context, r *pb.LeaseRevokeRequest) (*pb.LeaseRevokeResponse, error) {
+	if r.ID < int64(lease.NoLease) {
```

Comment:
```suggestion
	if r.ID <= int64(lease.NoLease) {
```

You should not be able to revoke NoLease